admin/grants.sql

```diff
@@ -505,6 +505,13 @@ GRANT ROLE SYNAPSE_DATA_WAREHOUSE_ANALYST TO ROLE SYNAPSE_DATA_WAREHOUSE_ADMIN;
 GRANT ROLE SYNAPSE_DATA_WAREHOUSE_DEV_ADMIN TO ROLE SYSADMIN;
 GRANT ROLE SYNAPSE_DATA_WAREHOUSE_DEV_ANALYST TO ROLE SYNAPSE_DATA_WAREHOUSE_DEV_ADMIN;
 
+-- CREATE DATABASE privileges on domain ADMIN roles
+USE ROLE ACCOUNTADMIN;
```

Comment:
I noticed that we execute `admin/grants.sql` using the `SNOWSQL_USER` secret. Are we sure that this user has ACCOUNTADMIN?

Since it was configured as a secret we have no way of telling which user is executing this part in the CI. @thomasyu888 ?

Reply:
It is currently using my user "thomasyu888" which should have accountadmin privileges